src/Relude/Extra/Bifunctor.hs

```diff
@@ -50,3 +51,19 @@ Just (False,2)
 secondF  :: (Functor f, Bifunctor p) => (b -> d) -> f (p a b) -> f (p a d)
 secondF = fmap . second
 {-# INLINE secondF #-}
+
+{- | Maps a function over both elements of a bifunctor.
```

Comment:
Let's move this function in before `bimapF` to be consistent with the order of functions in the export list.